201-servicebus-private-endpoint/README.md

```diff
@@ -0,0 +1,96 @@
+![Azure US Gov Last Test Result](https://azurequickstartsservice.blob.core.windows.net/badges/201-servicebus-private-endpoint/FairfaxDeployment.svg)
+
+![Best Practice Check](https://azurequickstartsservice.blob.core.windows.net/badges/201-servicebus-private-endpoint/BestPracticeResult.svg)
+![Cred Scan Check](https://azurequickstartsservice.blob.core.windows.net/badges/201-servicebus-private-endpoint/CredScanResult.svg)
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-servicebus-private-endpoint%2Fazuredeploy.json)
+[![Deploy To Azure US Gov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-servicebus-private-endpoint%2Fazuredeploy.json)
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-servicebus-private-endpoint%2Fazuredeploy.json)
+
+This sample demonstrates how to create a Linux Virtual Machine in a virtual network that privately accesses a Service Bus namespace using an [Azure Private Endpoint](https://docs.microsoft.com/en-us/azure/private-link/private-endpoint-overview). Azure Private Endpoint is a network interface that connects you privately and securely to a service powered by Azure Private Link. Private Endpoint uses a private IP address from your virtual network, effectively bringing the service into your virtual network. The service could be an Azure service such as Azure Storage, Azure Cosmos DB, SQL, etc. or your own Private Link Service. For more information, see [What is Azure Private Link?](https://docs.microsoft.com/en-us/azure/private-link/private-link-overview). For more information on the DNS configuration of a private endpoint, see [Azure Private Endpoint DNS configuration](https://docs.microsoft.com/en-us/azure/private-link/private-endpoint-dns).
+
+## Architecture ##
+
+The following picture shows the architecture and network topology of the sample.
+
+![Architecture](images/architecture.png)
+
+The ARM template deploys the following resources:
+
+- Virtual Network: this virtual network has a single subnet that hosts an Linux (Ubuntu) virtual machine
+- Network Security Group: this resource contains an inbound rule to allow the access to the virtual machine on port 22 (SSH)
+- The virtual machine is created with a managed identity which is assigned the contributor role at the resource group scope level

```

Comment:
why is this needed?

Reply:
Copy and paste from the README of a project on my GitHub account. I removed it.